.mailmap

```diff
@@ -1,135 +1,109 @@
+Mishal Shah <mishal_shah@apple.com> <mishal_shah@apple.com>
 Mishal Shah <mishal_shah@apple.com> <shahmishal@users.noreply.github.com>
-Nadav Rotem <nrotem@apple.com> <nadavrot@users.noreply.github.com>
-Nate Cook <natecook@apple.com> <nate@Nates-MacBook-Pro.local>
-Nate Cook <natecook@apple.com> <natecook@gmail.com>
-Nate Cook <natecook@apple.com> <natecook1000@users.noreply.github.com>
-Nate Cook <natecook@apple.com> <nmersethcook@apple.com>
-Nathan Lanza <lanza@fb.com> <nathan@lanza.io>
-Nicole Jacque <jacque@apple.com>
-Niels Andriesse <andriesseniels@gmail.com> <nielsandriesse@users.noreply.github.com>
-Paul Meng <mno2@mno2.org> <mno2.csie@gmail.com>
+Nikola Lajic <n.lajic@gmail.com> <n.lajic@gmail.com>
+Nikola Lajic <n.lajic@gmail.com> <nikola.lajic@codecentric.de>
+Owen Voorhees <owen@squibner.com> <owen@squibner.com>
+Owen Voorhees <owen@squibner.com> <owenvoorhees@gmail.com>
+Pavel Yaskevich <pyaskevich@apple.com> <pyaskevich@apple.com>
 Pavel Yaskevich <pyaskevich@apple.com> <xedin@apache.org>
-Paweł Szot <pszot@pgs-soft.com>
-Paweł Szot <pszot@pgs-soft.com> <qwertyszot@gmail.com>
-Pete Cooper <peter_cooper@apple.com>
-Philip Ridgeway <pridgeway@vernier.com> <philip.ridgeway@gmail.com>
-Richard Wei <rxwei@apple.com> <rxwei@google.com>
-Rintaro Ishizaki <rishizaki@apple.com> <fs.output@gmail.com>
-Robert Widmann <rwidmann@apple.com> <devteam.codafi@gmail.com>
-Roman Levenstein <rlevenstein@apple.com> <swiftix@users.noreply.github.com>
-Ross Bayer <ross.m.bayer@gmail.com> <Rostepher@users.noreply.github.com>
-Russ Bishop <rbishopjr@apple.com> <russ@plangrid.com>
-Ryan Lovelett <ryan@lovelett.me> <RLovelett@users.noreply.github.com>
-Shawn Erickson <shawn.erickson@citrix.com> <shawnce@gmail.com>
-Slava Pestov <spestov@apple.com> <spestov@rad-main.corp.apple.com>
+Saleem Abdulrasool <compnerd@compnerd.org> <compnerd@compnerd.org>
+Saleem Abdulrasool <compnerd@compnerd.org> <abdulras@fb.com>
+Saleem Abdulrasool <compnerd@compnerd.org> <abdulras@google.com>
+Sergio Campamá <scampama@apple.com> <scampama@apple.com>
+Sergio Campamá <scampama@apple.com> <kaipi@copilot.money>
+Slava Pestov <spestov@apple.com> <spestov@apple.com>
 Slava Pestov <spestov@apple.com> <sviatoslav.pestov@gmail.com>
-Stephen Canon <scanon@apple.com>
-Stephen Canon <scanon@apple.com> <stephentyrone@gmail.com>
-Sukolsak Sakshuwong <sukolsak@gmail.com>
-Todd Fiala <tfiala@apple.com> <todd.fiala@gmail.com>
-Toni Suter <tonisuter@me.com> <tonisuter@users.noreply.github.com>
-Vedant Kumar <vsk@apple.com> <vk@vedantk.com>
-Xi Ge <xi_ge@apple.com> <xi_ge@rad-main.corp.apple.com>
-Xin Tong <xin_tong@apple.com> <trent.xin.tong@gmail.com>
-Xin Tong <xin_tong@apple.com> <trentxintong@Xins-MacBook-Pro.local>
-Yuka Ezura <ezura@users.noreply.github.com> <2020337+ezura@users.noreply.github.com>
-Yurii Samsoniuk <ura@google.com> <mr.sigito@gmail.com>
-Zac Bowling <zbowling@google.com> <zac@zacbowling.com>
-冀卓疌 <55120045+WowbaggersLiquidLunch@users.noreply.github.com>
```

Comment:
Sorry for commenting after the PR's been merged. 

I notice that my information has been dropped from `.mailmap`, and I'm mentioned on `CONTRIBUTORS.txt` twice, once by my old pseudonym, once by real name. Not sure if the changes are intended.